### PR TITLE
C solution 3 Added striped blocks

### PR DESCRIPTION
**Pull request by fvbakel**
## Description
C solution 3 Added striped blocks, based on [Rust solution 1](../../PrimeRust/solution_1) by @mike-barber. It does not match that performance but still I would like to add it because this technique is not implemented in C yet.

## Contributing requirements
* [X] I read the contribution guidelines in CONTRIBUTING.md.
* [X] I placed my solution in the correct solution folder.
* [X] I added a README.md with the right badge(s).
* [X] I added a Dockerfile that builds and runs my solution.
* [X] I selected `drag-race` as the target branch.
* [X] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeC/solution_3/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 primes_words
+primes_stripped-block
```
Comment (rbergen) [inline PrimeC/solution_3/.gitignore]: I'm only going to point it out here, but you may have intended this to be `primes_striped-block` (minus one p) throughout your solution.

**Review by rbergen — APPROVED** [review]
There's one small textual thing you may want to change (see below), but even in its current for this LGTM.

File: PrimeC/solution_3/.gitignore (hunk 1 above)
Comment (fvbakel) [reply PrimeC/solution_3/.gitignore]: @rbergen Thank you for noticing. Al least it was a consistent copy paste typo in al places :-)

Comment (rbergen) [reply PrimeC/solution_3/.gitignore]: @fvbakel Yes, I did notice that :D

**Comment by fvbakel** [comment]
@rbergen please wait with the merge, I just discovered that there is a issue with the performance when I run from Docker. I want to double check that for the merge.

**Comment by fvbakel** [comment]
@rbergen I make one small change, I changed the storage to uint32_t instead of char because that seems to make a big difference in the performance in docker. The docker performance was 1500 passes in 5 seconds with this small update it becomes 9832 passes in 5 seconds!. 

It is now ready for the merge :-)

**Review by rbergen — APPROVED** [review]

**Review by marghidanu — APPROVED** [review]

**Pull request merged**